Sources/PackagePlugin/Protocols.swift

```diff
@@ -62,7 +74,35 @@ public protocol CommandPlugin: Plugin {
 }
 
 extension CommandPlugin {
-    
+    /// Compatibility implementation that calls the older version of this method.
+    public func performCommand(
```

Comment:
if we do this default implementation, then the protocol requirement will do nothing for plugins, ie they will not know which api they need to implement. i think we should just delete the one with the targets and not have a default

Reply:
Yes, this is one of the problems.  I agree that we should remove it.